src/components/PeerGroupsUpdate.tsx

```diff
@@ -0,0 +1,193 @@
+                    </svg>
+                </Col>
+            </Row>
+        </>
+    )
+
+    const setUpdateGroupsVisible = (status:boolean) => {
+        dispatch(peerActions.setUpdateGroupsVisible(status));
+    }
+
+    const onCancel = () => {
+        dispatch(peerActions.setPeer(null))
+        setUpdateGroupsVisible(false)
+    }
+
+    const onChange = (data:any) => {
+        //setFormRule({...formRule, ...data})
+    }
+
+    const handleChangeTags = (value: string[]) => {
+        setSelectedTagGroups(value)
+    };
+
+    const handleFormSubmit = () => {
+        form.validateFields()
+            .then((values) => {
+                dispatch(peerActions.saveGroups.request({getAccessTokenSilently, payload: peerGroupsToSave}))
+            })
+            .catch((errorInfo) => {
+                console.log('errorInfo', errorInfo)
+            });
+    }
+
+    return (
+        <>
+            {peer &&
+                <Drawer
+                    title={`${peer.Name}`}
+                    forceRender={true}
+                    visible={true}
+                    bodyStyle={{paddingBottom: 80}}
+                    onClose={onCancel}
+                    autoFocus={true}
+                    footer={
+                        <Space style={{display: 'flex', justifyContent: 'end'}}>
+                            <Button onClick={onCancel} disabled={savedGroups.loading}>Cancel</Button>
+                            <Button type="primary" disabled={savedGroups.loading} onClick={handleFormSubmit}>Save</Button>
+                        </Space>
+                    }
+                >
+                    <Form layout="vertical" hideRequiredMark form={form} onValuesChange={onChange}>
+                        <Row gutter={16}>
+                            <Col span={24}>
+                                <Form.Item
+                                    name="groups"
+                                    label="Groups"
+                                    rules={[{required: true, message: 'Please enter ate least one group'}]}
+                                    style={{display: 'flex'}}
+                                >
+                                    <Select mode="tags"  style={{ width: '100%' }} placeholder="Select groups..." tagRender={tagRender} dropdownRender={dropDownRender} onChange={handleChangeTags}>
+                                        {
+                                            tagGroups.map(m =>
+                                                <Option key={m}>{optionRender(m)}</Option>
+                                            )
+                                        }
+                                    </Select>
+                                </Form.Item>
+                            </Col>
+                        </Row>
+                    </Form>
+                </Drawer>
```

Comment:
Codacy has a fix for the issue: [Replace `················` with `········`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
        </Drawer>
```